### DISCUSSION TITLE
Expose Ref<T> type

### DISCUSSION THREAD

**Discussion by elite174** [solidjs/solid#905] [Ideas]
I suggest to expose Ref<T> from the lib, because it can be useful sometimes to use this type instead of writing `ref: T | (element: T) => void`

**Comment by elite174** [comment]
Oh, I've just seen this PR https://github.com/solidjs/solid/pull/877

Does it expose `Ref` type? If so, can it be merged then?

**Reply by lxsmnsyc** [reply]
I think the new types are going in for the next minor release due to some type changes.
